raddb/mods-available/rest

```diff
@@ -98,16 +98,24 @@ rest {
 	#  [outer.][<list>:]<attribute0>=<value0>&[outer.][<list>:]<attributeN>=<valueN>
 	#
 	#  JSON - All attributes and values are escaped according to the JSON specification
+	#  - attribute  Name of the attribute.
+	#  - attr_type  Numerical representation of the attribute type.
```

Comment:
can you rename to `attr_num` and place inside a `json { }` subsection within the method subsection.

`attr_type` is RADIUS specific terminology, and FreeRADIUS isn't a RADIUS server.  It just happens to have a RADIUS frontend :)